docs/how-to-open-a-pull-request.md

```diff
@@ -2,6 +2,9 @@
 
 A pull request (PR) enables you to send changes from your fork on GitHub to freeCodeCamp.org's main repository. Once you are done making changes to the code, you can follow these guidelines to open a PR.
 
+> [!NOTE]
+> Your pull request title, body, commit messages, and code changes (including variable names and comments) should all be in English.
```

Comment:
```suggestion
> Your PR should be in English. See [here](#translations) for how to contribute translations.
```
It's a bit terse, but I think it gets the point across okay. What do you reckon?

Reply:
Terse is good, but I wonder if it's clear enough or if "PR" is too ambiguous here.

Reply:
It might be, yeah.  It's tricky with docs. Your way was clearer, but the more you ask people to read, the more likely it is they just don't bother.  I'm not sure what the optimal length is, though.

Reply:
We can certainly try it this way - if we get a rush of non-english PRs or commit messages we can always re-evaluate.